app/views/projects/_form.slim

```diff
@@ -61,6 +61,19 @@
               |  and,
               tt description
               |  .
+        .control-group
+          = f.label :article_webhook_url, class: 'control-label'
+          .controls
+            = f.text_field :article_webhook_url, class: 'tooltip-parent', :'data-tooltip' => '#article_webhook_url_tooltip'
+            .tooltip#article_webhook_url_tooltip
+              ' This URL will be posted to when an article's translation gets ready.
+                It is used to notify client systems about translations getting available to be consumed.  HTTP Post body will contain
+              tt article_name
```

Comment:
can improve readability using inline HTML `<tt>...</tt>` if you'd like

Reply:
Think I will let it this way, to keep consistent with the code of the other tooltips in the page.